README.md

```diff
@@ -20,6 +20,27 @@ Keep in mind that:
 * There is another repo containing the [deprecated version](https://github.com/microg/NetworkLocation) of NetworkLocation.apk without the plug-in system.
 * [microG GmsCore](https://github.com/microg/android_packages_apps_GmsCore/wiki) project already includes the Unified Network Location Provider.
 
+### Android 9
+Since google is now enforcing permission-whitelisting for system apps just copying the apk to `/system/priv-app/` is not sufficient anymore. Therefore an xml file with the following contents has to be placed at `/system/etc/permissions`:
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<permissions>
+    <privapp-permissions package="com.google.android.gms">
+        <permission name="android.permission.INSTALL_LOCATION_PROVIDER"/>
+        <permission name="android.permission.ACCESS_FINE_LOCATION"/>
+        <permission name="android.permission.ACCESS_COARSE_LOCATION"/>
+        <permission name="android.permission.ACCESS_COARSE_UPDATES"/>
+    </privapp-permissions>
+</permissions>
+```
+This leaves us with the following installation steps:
+
+1. Make sure no gapps are installed
+2. Download `NetworkLocation.apk`
+3. Boot into recovery and copy the apk to `/system/priv-app/` using the adb-command `adb push NetworkLocation.apk /system/priv-app/UnifiedNLP/NetwortkLocation.apk` (also make sure you mount the `/system` partition in advance)
```

Comment:
I'm not sure if it really matters for the functionality, but it'd be nice to update `NetwortkLocation.apk` to be spelled correctly

Otherwise, it'd be great to see this merged - if this was already in the README a few months ago I wouldn't have had to embarrass myself a few times by not having working GPS on my phone :sweat_smile:

Reply:
> I'm not sure if it really matters for the functionality[...]

Well, at least in my case, the app didn't registered itself as system-wide location provider (Android 9, LineageOS) when installed as a "normal" app.

> but it'd be nice to update NetwortkLocation.apk to be spelled correctly.

Done, thanks :+1: 

> Otherwise, it'd be great to see this merged

Actually I don't know what @mar-v-in 's plan is here - there seems to be active development though...